src/DefaultRoute.ts

```diff
@@ -0,0 +1,38 @@
+import Promise from 'dojo-core/Promise';
+import PathRule from './PathRule';
+import RouteManager from './RouteManager';
+import { CancelableNavigationArgs, DefaultRouteArgs, MatchableRoute, NavigationArgs } from './routing';
+
+export default class DefaultRoute implements MatchableRoute {
+	protected _rule: PathRule;
+
+	beforeEnter: (kwArgs: CancelableNavigationArgs) => (Promise<void> | void);
```

Comment:
I thought we were going to try to drop `kwArgs` in lieu of `options`, @kfranqueiro?  Maybe we should add it to the style guide?